src/CoreShop2VueStorefrontBundle/Bridge/ImporterFactory.php

```diff
@@ -26,9 +26,9 @@ public function __construct(PersisterFactory $persisterFactory)
     /**
      * @return array<ImporterFactory>
      */
-    public function create(?string $site = null, ?string $type = null, ?string $language = null, ?string $store = null, ?\DateTimeInterface $since = null): array
+    public function create(?string $site = null, ?string $type = null, ?string $language = null, ?string $store = null, ?\DateTimeInterface $since = null, ?string $runTimestamp = null): array
```

Comment:
We don't need this to be configurable, it will never be changed to anything else.